main.js

```diff
@@ -0,0 +1,315 @@
+console.log(circlesAreaSum);
+
+// 7
+
+function convertMinutesToHours (minutesAmount) {
+    return minutesAmount / 60;
+}
+
+const hours = convertMinutesToHours(75);
+console.log(hours);
+
+// 8
+
+function isTextUppercase(text) {
+    if (text === text.toUpperCase()) {
+        return true;
+    }
+    return false;
+}
+
+console.log(isTextUppercase('Hello'));
+console.log(isTextUppercase('HELLO'));
+
+// 9
+
+function isDivisible(n, x, y) {
+    if (n % x === 0 && n % y === 0) {
+        return true;
+    }
+    return false;
+}
+
+console.log(isDivisible(5, 1, 5));
+console.log(isDivisible(5, 1, 3));
+
+// 10
+
+function getBillboardPrice (billboardText, singleCharacterCost) {
+    return billboardText.length * singleCharacterCost;
+}
+
+console.log(getBillboardPrice('Hello world!', 10));
+console.log(getBillboardPrice('Hello world!', 15));
+console.log(getBillboardPrice('To be, or not to be', 20));
+
+// 11
+
+function getOppositeNumber(number) {
+    return number * -1;
+}
+
+console.log(getOppositeNumber(1));
+
+// 12
+
+function getNegativeNumber(num) {
+    if (num <= 0) {
+        return num;
+    }
+    return num * -1;
+}
+
+console.log(getNegativeNumber(5));
+console.log(getNegativeNumber(0));
+console.log(getNegativeNumber(-3));
+
+// 13
+
+function removeFirstAndLastCharacter(str){
+    return str.slice(1, -1);
+}
+
+console.log(removeFirstAndLastCharacter("Amazing"));
+
+// 14
+
+function getBasicOperationsResult(operation, value1, value2){
+    if (operation === '+') {
+        return value1 + value2;
+    } else if (operation === '-') {
+        return value1 - value2;
+    } else if (operation === '*') {
+        return value1 * value2;
+    }
+    return value1 / value2;
+}
+
+console.log(getBasicOperationsResult("+", 5, 10));
+
+// 15
+
+function whichCentury(year) {
+    return Math.ceil(year / 100);
+}
+
+console.log(whichCentury(301));
+
+// 16
+
+function isEvenOrOdd(number) {
+    if (number % 2 === 0) {
+        return "Even";
+    }
+    return "Odd";
+}
+
+// 17
+
+function areYouPlayingBanjo(name) {
+    if (name[0] === 'R' || name[0] === 'r') {
+        return name + ' plays banjo';
+    }
+    return name + ' does not play banjo'
+}
+
+// 18
+
+function bmi(weight, height) {
+    const bodyMassIndex = weight / (height * height);
+
+    switch (true) {
+        case (bodyMassIndex <= 18.5):
+            return "Underweight";
+        case (bodyMassIndex <= 25.0):
+            return "Normal";
+        case (bodyMassIndex <= 30.0):
+            return "Overweight";
+        default:
+            return "Obese";
+    }
+}
+
+console.log(bmi(20, 1.80));
+
+// 19
+
+function rockPaperScissors (player1, player2) {
+    if (player1 === player2) {
+        return 0;
+    } else if ((player1 === "rock" && player2 === "scissors") ||
+    (player1 === "scissors" && player2 === "paper") ||
+    (player1 === "paper" && player2 === "rock"))
+    {
+        return 1;
+    }
+    return 2;
```

Comment:
This does not look good. Try to clean this code up. Try declaring additional variables.

Reply:
Any hint?